src/auro-flightline.js

```diff
@@ -21,6 +21,14 @@ import styleCss from "./style-flightline-css.js";
 // build the component class
 class AuroFlightline extends LitElement {
 
+    // function to define props used within the scope of this component
+    static get properties() {
+      return {
+        // ...super.properties,
+        canceled:   { type: Boolean }
```

Comment:
```suggestion
        canceled:   { 
          type: Boolean,
          reflect: true
         }
```

https://lit-element.polymer-project.org/guide/properties#reflected-attributes

It is most likely that this will be programmatically changed.

Reply:
@blackfalcon I don't think this is necessary since we're not using the attribute as a styling hook, i.e. `:host([canceled])`. Or is there another reason you want the property reflected?

Reply:
My reasoning was that if this were to be updated programmatically... and it's become somewhat of a reflex.